lib/prom_ex/dashboard_uploader.ex

```diff
@@ -168,7 +168,7 @@ defmodule PromEx.DashboardUploader do
       {:ok, folder_details} ->
         folder_details
 
-      {:error, :bad_request} ->
+      {:error, _} ->
```

Comment:
Perhaps some logging can be added here in case there is an error that requires users to act upon it.

Reply:
ok I've inserted a Logger error message that matches the style of existing log messages. I think it should be clear enough what happened and the expectation is that the `"There was a folder UID mismatch for the folder titled \"#{folder_name}\". PromEx has updated the folder configuration in Grafana and resolved the issue."` message will be fire after prom_ex recovers from the issue.